.github/workflows/release-30_publish_release_draft.yml

```diff
@@ -26,6 +29,15 @@ jobs:
     uses: "./.github/workflows/release-srtool.yml"
     with:
        excluded_runtimes: "substrate-test bp cumulus-test kitchensink minimal-template parachain-template penpal polkadot-test seedling shell frame-try sp solochain-template"
+       build_opts: "--features on-chain-release-build"
+       name_suffix: ""
+
+  build-runtimes-with-logging:
+    uses: "./.github/workflows/release-srtool.yml"
+    with:
+       excluded_runtimes: "substrate-test bp cumulus-test kitchensink minimal-template parachain-template penpal polkadot-test seedling shell frame-try sp solochain-template"
+       build_opts: ""
```

Comment:
This doesn't enable the `on-chain-release-build` feature. This means that metadata hash will not be generated at compile time. This is bad :see_no_evil:

Reply:
Yes, this one was done only for the case to have the logging enabled. In theory, this step should go away if the `disable-logging` feature will be deleted from the `on-chain-release-build` as we discussed above. If it is ok, I can delete it as part of this PR, and then we will have only the one build with the `on-chain-release-build` feature activated.